scss/_code.scss

```diff
@@ -17,13 +17,11 @@ kbd {
   color: $kbd-color;
   background-color: $kbd-bg;
   @include border-radius($border-radius-sm);
-  @include box-shadow($kbd-box-shadow);
 
-  kbd {
+  & {
```

Comment:
maybe we should revert this to `kbd` so that stylelint passes or ignore the warning?